pkg/volume/plugins.go

```diff
@@ -334,6 +334,8 @@ type KubeletVolumeHost interface {
 	CSIDriversSynced() cache.InformerSynced
 	// WaitForCacheSync is a helper function that waits for cache sync for CSIDriverLister
 	WaitForCacheSync() error
+	// Returns HostUtils Interface
+	GetHostUtil() mount.HostUtils
```

Comment:
Is this used?

Reply:
Yes. In the other place you commented on -- it's used throughout various volume plugins.